### PR TITLE
Fix transactions

### PR DESCRIPTION
**Pull request by msteiner96**
Removed chainId from query, because it's not needed here. Addresses will most probably be unique anyways. Also stopped querying w/o a address.

Should work :)

### CODE REVIEW

File: lib/graphqlHelpers.js [hunk 1]
```diff
@@ -49,7 +48,7 @@ const getMultisig = async (address, chainId) => {
     data: {
       query: `
         query {
-          getMultisig(address: "${address}", chainId: "${chainId}",) {
+          getMultisig(address: "${address}") {
```
Comment (samepant) [inline lib/graphqlHelpers.js:51]: Although unlikely, it is possible for an address to be the same across chains, so I'd rather be prepared than have it break if someone attempts to use it in that way.

**Review by samepant — CHANGES_REQUESTED** [review]
I'm ok with this PR if 7fea0b7e752b8fa7d83a726845946f8693b89a60 is removed and the `useEffect` change combined with the other call.

File: lib/graphqlHelpers.js (hunk 1 above)
Comment (msteiner96) [reply lib/graphqlHelpers.js:51]: Hi, thanks for the quick feedback! It was impossible for me to query chainid with the provided graphql scheme, that's why I changed it.

Comment (samepant) [reply lib/graphqlHelpers.js:51]: What issue were you running into?

Comment (msteiner96) [reply lib/graphqlHelpers.js:51]: I just couldn't query the chainid, even tried on faunadbs playground. That's why I got errors

Comment (msteiner96) [reply lib/graphqlHelpers.js:51]: Probably you have a different scheme uploaded?

Comment (samepant) [reply lib/graphqlHelpers.js:51]: this is the schema thats used https://github.com/cosmos/cosmos-multisig-ui/blob/master/db-schema.graphql

Good catch! i need to update the query in that schema file to include the chainId

Comment (webmaster128) [reply lib/graphqlHelpers.js:51]: @samepant feel free to fix this in your PR branch and we close here. Then we can continue testing tomorrow.

Comment (webmaster128) [reply lib/graphqlHelpers.js:51]: @msteiner96 is a buddy from Confio by the way

**Comment by samepant** [comment]
ok fixed both issues in #24

**Pull request closed without merging**